nodes/alarm_server.py

```diff
@@ -97,7 +101,7 @@ def update(self, srv):
         self.stamp = rospy.Time.now()
         
         node_name = "unknown" if srv.node_name is "" else srv.node_name
-        parameters = self.parse_json_str(srv.parameters)
+        parameters = '' if srv.parameters is '' else json.loads(srv.parameters)
```

Comment:
Oh, whoops. I didn't mean to change that (not sure how that happened...).
Okay, I was wondering how to run those - I will make sure to do that from now on!